.github/workflows/ingest-pull.yml

```diff
@@ -1,17 +1,24 @@
 
 name: ingest pull
 
-
 on:
-  pull_request:
-
+  pull_request_target:
+    types: [labeled]
 
 env:
   NODE_VERSION: 14.17
 
-
 jobs:
+  security:
+    name: Check permissions for pull request runs
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'safe to test')
```

Comment:
what happens if label isn't found? does the action stops?

Reply:
I think so https://securitylab.github.com/research/github-actions-preventing-pwn-requests/